main.go

```diff
@@ -212,12 +243,24 @@ func runCopy(db *redshift.Redshift, inputConf s3filepath.S3File, inputTable reds
 	return nil
 }
 
-func startEndFromGranularity(t time.Time, granularity string) (time.Time, time.Time) {
+func startEndFromGranularity(t time.Time, granularity string, isTargetDataPT bool) (time.Time, time.Time) {
+	// Rotate time if in PT
+	if isTargetDataPT {
+		ptLoc, err := time.LoadLocation("America/Los_Angeles")
+		fatalIfErr(err, "startEndFromGranularity was unable to load PT timezone")
+
+		_, ptOffsetSec := t.In(ptLoc).Zone()
+		ptOffsetDuration, err := time.ParseDuration(fmt.Sprintf("%vs", ptOffsetSec))
+		fatalIfErr(err, "startEndFromGranularity was unable to parse offset duration")
+
+		t = t.Add(ptOffsetDuration)
+	}
+
 	var duration time.Duration
 	if granularity == "day" {
 		duration = time.Hour * 24
 	} else {
-		duration = time.Hour * 24 * 7
```

Comment:
why was the default a week previously? Was the default used anywhere? We're changing to hourly as a default here...